projects/v2/src/app/app-init.service.ts

```diff
@@ -0,0 +1,20 @@
+import { Injectable }  from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+ 
+@Injectable()
+export class AppInitService {
+
+    SHEET_CONFIGURATION:any
+ 
+    constructor(private readonly http: HttpClient) {
+    }
+    
+    Init() {
```

Comment:
This is still yet another band-aid, but for the sake of time I'll allow it. However, please refactor this further to be "SheetConfigService" and have this be the central service for getting sheet configuration. I do not want to see that config.ts in existence anymore.